frontend/src/metabase/plugins/index.ts

```diff
@@ -225,7 +225,7 @@ const defaultLoginPageIllustration = {
   isDefault: true,
 };
 
-const getLoadingMessage = (isSlow: boolean) =>
+const getLoadingMessage = (isSlow?: boolean) =>
```

Comment:
the nittiest of nits
```suggestion
const getLoadingMessage = (isSlow: boolean = false) =>
```